### DISCUSSION TITLE
Navigate to ... Action 1.Saved Row

### DISCUSSION THREAD

**Discussion by JeroenP1974** [Budibase/budibase#12995] [Help] [answered]
Good day. I would like some assistance with a problem.

I have one table for data. And I have two screens. One I use for adding data and the other I want to use to view the data I just added. I tried with navigate to and action 1.saved row._id. But when I push the button it is not going to the wanted screen.

I use screen /invoegen for adding data. And after pushing the button I want it to goto screen /namen/:id with de latest data/record I added. I can't figure out what I'm doing wrong. Thanks for your help!

<img width="1131" alt="Scherm­afbeelding 2024-02-09 om 01 24 55" src="https://github.com/Budibase/budibase/assets/157612513/bec2b0e2-6b1f-420e-a997-223157c28825">

**Comment by melohagan** [comment, marked as answer]
Hey @JeroenP1974 

I think this is related to this known bug: https://github.com/Budibase/budibase/issues/12965

**Reply by JeroenP1974** [reply]
Thanks!
